worker/src/handlers/schedule.ts

```diff
@@ -69,18 +70,43 @@ async function resetQueue(sentry: Toucan): Promise<void> {
   }
   sentry.addBreadcrumb({ message: "Process complete" });
 }
+
 async function updateHistory(sentry: Toucan): Promise<void> {
   sentry.setTag("scheduled-task", "UPDATE_HISTORY");
   sentry.addBreadcrumb({ message: "Process started" });
   let data = createQueueData();
 
   sentry.addBreadcrumb({ message: "Get current data" });
   const analyticsData = await getAnalyticsData();
-  const timestampString = String(new Date().getTime());
+  const timestamp = new Date().getTime();
+  const timestampString = String(timestamp);
 
   sentry.addBreadcrumb({ message: "List UUID entries" });
   const kv_list = await listKV(KV_PREFIX_UUID);
 
+  const missingMetata = kv_list.filter((entry) => !entry.metadata);
+
+  async function handleMissingMetadata(entry: ListEntry) {
+    try {
+      const value = await KV.get<IncomingPayload>(entry.name, "json");
+      await KV.put(entry.name, JSON.stringify(value), {
+        expiration: entry.expiration,
+        metadata: generateUuidMetadata(value!, timestamp),
+      });
+    } catch (e) {
+      sentry.addBreadcrumb({ message: e });
```

Comment:
Since we never log an error, the  breadcrumbs will never be published to sentry. Maybe better to return them and then log at least one of them?

Reply:
This is only here to provide more context if something fails later (same as we do for the process queue task)

Reply:
isn't that a bit weird, we might be missing errors in that case ?

Reply:
REmoved the try block, so it will fail hard on any issue